app/assets/stylesheets/screens/_keywords.scss

```diff
@@ -0,0 +1,17 @@
+.keywords-table {
+  @extend .border;
+  max-height: 75vh;
+  overflow: auto;
+
+  &__body {
+    font-variant-numeric: tabular-nums;
+  }
+}
+
+/* stylelint-disable selector-max-id */
```

Comment:
The reason for this is: pagy only allows us to customize id, not class and I don't to wrap the pagination in another div.

Reply:
> And I don't to wrap the pagination nav in another div.

Wrong call 😅 It would be best to have another `div` to isolate a third-party dependency.

Reply:
Fixed in 5b30f0b3e4cb3fbd986088e80091cb1259cb6184